tuf/client_rework/updater_rework.py

```diff
@@ -35,27 +36,32 @@ class Updater:
     """
     Provides a class that can download target files securely.
 
-    Attributes:
-        metadata:
-
-        repository_name:
-
-        mirrors:
-
-        fetcher:
-
-        consistent_snapshot:
+    TODO
     """
 
     def __init__(
         self,
         repository_name: str,
-        repository_mirrors: Dict,
+        metadata_base_url: str,
+        target_base_url: Optional[str] = None,
         fetcher: Optional[FetcherInterface] = None,
     ):
-
+        """
+        Args:
+            repository_name: directory name (within a local directory
+                defined by 'tuf.settings.repositories_directory')
```

Comment:
This took me a couple of attempts to parse, I'm not sure my suggestion is any better though.
```suggestion
            repository_name: name of the directory used for storing local copies of working files
                for this repository. Created as a child of 'tuf.settings.repositories_directory'.
```

Reply:
This is better but I'm not changing it as I intend to
* not use tuf.settings for anything at all in near future
* use local_repo_dir as the argument directly -- there's no need for a system wide repositories_directory (at least not one managed by Updater)
this will happen after we have a MetadataBundle #1355 we can use and should be much easier to understand and document